components/rmrk/Create/3DViewer/ThreeDViewer.vue

```diff
@@ -0,0 +1,245 @@
+<template>
+  <div id="canvasContainer" class="canvasContainer"></div>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue, Watch } from 'nuxt-property-decorator'
+import {
+  AmbientLight,
+  Box3,
+  Cache,
+  DirectionalLight,
+  Light,
+  LinearToneMapping,
+  Object3D,
+  PerspectiveCamera,
+  Scene,
+  Vector3,
+  WebGLRenderer,
+  sRGBEncoding,
+} from 'three'
```

Comment:
Cool, I'll get on it. Thank you

Reply:
@preschian 
I've deleted my ThreeDViwer component and the three dependency
using the existing 'ModelMedia' component to display the model
the existing 'ar-button' in ModelMedia swallowed click event, preventing the file upload menu from popping up
to solve that I added a props 'withARButton' to ModelMedia, the prop default value is true, to avoid breaking previous behavior

waiting for you review
Thanks